Cleanup slf4j providers in the itest

The itests don't log correctly due to:

```
[INFO] [INFO] Running org.apache.karaf.camel.itest.CamelEhcacheITest
[ERROR] SLF4J(W): Class path contains multiple SLF4J providers.
[ERROR] SLF4J(W): Found provider [org.ops4j.pax.logging.slf4j.PaxLoggingSLF4JServiceProvider@59e5ddf]
[ERROR] SLF4J(W): Found provider [ch.qos.logback.classic.spi.LogbackServiceProvider@536aaa8d]
[ERROR] SLF4J(W): See https://www.slf4j.org/codes.html#multiple_bindings for an explanation.
[ERROR] SLF4J(I): Actual provider is of type [org.ops4j.pax.logging.slf4j.PaxLoggingSLF4JServiceProvider@59e5ddf]
```

We should have a clean provider here. I gonna fix that.

--- a/tests/examples/pom.xml
+++ b/tests/examples/pom.xml
@@ -120,6 +120,10 @@
                     <groupId>org.apache.karaf</groupId>
                     <artifactId>org.apache.karaf.client</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.ops4j.pax.logging</groupId>
+                    <artifactId>pax-logging-logback</artifactId>
+                </exclusion>
             </exclusions>
         </dependency>
         <!-- Required to use shell commands in the tests -->

--- a/tests/features/pom.xml
+++ b/tests/features/pom.xml
@@ -186,6 +186,10 @@
                     <groupId>org.apache.karaf</groupId>
                     <artifactId>org.apache.karaf.client</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.ops4j.pax.logging</groupId>
+                    <artifactId>pax-logging-logback</artifactId>
+                </exclusion>
             </exclusions>
         </dependency>
         <!-- Required to use shell commands in the tests -->

